src/ntimporters/monday/monday_api.py

```diff
@@ -50,19 +50,20 @@ def tasks(self, project_id: str) -> list:
         tasks = []
         for task in self._req(query).get("data", {}).get("boards", [{}])[0].get("items", []):
             counter, due_at = 0, None
+            task["is_all_day"] = False
             for column in task.get("column_values"):
-                if column.get("type") == "date" and column.get("value"):
+                if column.get("type") == "date" and column.get("text"):
                     if (counter := counter + 1) > 1:
                         break
                     try:
-                        dtime = json.loads(column.get("value", "{}"))
+                        dateformat = "%Y-%m-%d %H:%M"
+                        if len(column.get("text", "")) == 10:
+                            task["is_all_day"], dateformat = True, "%Y-%m-%d"
```

Comment:
you could just perform `parse_timestamp(column.get("text"))`, it should handle both formats automatically